src/power_meter.h

```diff
@@ -0,0 +1,74 @@
+#ifndef __POWER_METER_H__
+#define __POWER_METER_H__
+
+#include <ModbusMaster.h>
+
+// Implemented for use with PZEM-014
+
+class PowerMeter {
+ public:
+  PowerMeter(uint8_t device_address, int pin_de, int pin_re);
+  ~PowerMeter();
+
+  // TODOLater - Determine if calibration function is required
+  // TODOLater - Determine if reset energy function is required
```

Comment:
Why would we want to reset the energy readings? What does it even mean to reset a measurement in this context?